src/RocketChatMessage.php

```diff
@@ -1,10 +1,8 @@
 <?php
 
-declare(strict_types=1);
-
 namespace NotificationChannels\RocketChat;
 
-final class RocketChatMessage
```

Comment:
Would not recommend final. We don't know what the user does, I see no problem with allowing to inherit from that class

Reply:
If it will be a problem - users will write to us and we can remove final and it wouldn't be a breaking change. But you cannot do opposite thing without breaking changes.